client/landing/gutenboarding/onboarding-block/style-preview/style.scss

```diff
@@ -29,16 +29,24 @@
 	align-self: start;
 }
 
-// Extra specificity to override core component border
-.style-preview__font-option.components-button {
-	border: 1px solid var( --studio-gray-10 );
+.style-preview__font-option {
+	// Extra specificity to override core style
+	&.components-button {
```

Comment:
Just a tiny nit; styles `min-height`, `box-shadow`, `justify-content` etc should be before this line as is a common pattern.  I think we've had a lint for it in the past...